lib/recommended.js

```diff
@@ -73,10 +73,46 @@ module.exports = {
         es6: true,
     },
     rules: {
+        // @typescript-eslint rules
         "@typescript-eslint/no-unused-vars": [
             "error",
             { argsIgnorePattern: "^_", varsIgnorePattern: "^_" },
         ],
+        "@typescript-eslint/no-unnecessary-condition": "error",
+        "@typescript-eslint/no-floating-promises": "warn",
+
+        // react rules
+        "react/jsx-boolean-value": ["error"],
+        "react/self-closing-comp": ["error"],
+        "react/jsx-no-useless-fragment": ["error", { allowExpressions: true }],
+        "react/jsx-key": ["error"],
+        "react/jsx-sort-props": [
+            "error",
+            {
+                callbacksLast: true,
+                shorthandFirst: true,
+                reservedFirst: true,
+            },
+        ],
+
+        // stock ESLint
+        "padding-line-between-statements": [
+            "error",
+            {
+                blankLine: "always",
+                prev: [
+                    "multiline-block-like",
+                    "multiline-expression",
+                    "multiline-const",
+                ],
+                next: "*",
+            },
+        ],
```

Comment:
Looks like this was [deprecated in in ESLint v8.53.0](https://eslint.org/docs/latest/rules/padding-line-between-statements) and moved to [Eslint Stylistic](https://eslint.style/rules/js/padding-line-between-statements). I like the idea of this rule though, should we look into adding [the  plugin](https://eslint.style/guide/getting-started)?